### ISSUE TITLE
[Discussion] Next steps

### ISSUE THREAD

**Issue by Ash258** [lukesampson/scoop#3393]

**Comment by h404bi** [comment]
https://github.com/scoopinstaller/install is blocked by #3242 and #3121

**Comment by Ash258** [comment]
I forgot one main step :O 

🎉🎉 Finally delete bucket folder from this repo after develop is merged to master 🎉🎉

**Comment by r15ch13** [comment]
E-Mail @lukesampson about the projects feature. :smile:

**Comment by d2s** [comment]
> E-Mailed lukesampson about the projects feature. 😄

There is now a Projects section on the Scoop GitHub repository:
https://github.com/lukesampson/scoop/projects

Apparently the repository admins / leading contributors have the required permissions to use it?

**Comment by kiedtl** [comment]
hello,

I really like Scoop and have been using it for several years - I'd like to help out now. Anything I can do?

**Comment by Jogai** [comment]
I vote for a proper changelog. Its nice to have and can be used to announce releases on third-party sites. (I submit to a cerntain site that keeps track of some releases, but it requires a unique link for each version).
